core/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -72,9 +72,30 @@ abstract class AppBase(
   def getOrCreateStage(info: StageInfo): StageInfoClass = {
     val stage = stageIdToInfo.getOrElseUpdate((info.stageId, info.attemptNumber()),
       new StageInfoClass(info))
+
     stage
   }
 
+  def checkMLOps(stageInfo: String): Array[String] = {
```

Comment:
we plan to use this with profiling tool as well?  otherwise put into QualificaitonAppInfo with getMlFunction

Reply:
Yes, plan is to use it with profiling tool as well later.